orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/files/self_sign_certs.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
I see that this is being duplicated. Can we use the original script instead so that we don't go out of sync going forward.

Reply:
@karthiksubraveti Ansible doesn't really like this file if it's a symlink. I'm open to some ideas if you have any

Reply:
May be i'm missing something here. If we provide MAGMA_ROOT as an input variable, can't we reference the script under orc8r directory here 
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/script_module.html ?